docs/Guides/Recommendations.md

```diff
@@ -298,3 +299,36 @@ readinessProbe:
     timeoutSeconds: 3
     successThreshold: 1
     failureThreshold: 5
+```
+
+## Capacity planning for production
+<a id="capacity"></a>
+
+In order to rightsize your fastify application deployment in production, 
+you are highly recommended to perform your own measurements against 
+different configurations of the production environment 
+(2 vCPU, 1 vCPU, fractional parts of vCPU).
+
+You can use such tools as [k6](https://github.com/grafana/k6) 
+or [autocannon](https://github.com/mcollina/autocannon) for conducting
+the necessary performance tests.
+
+That said, you may also consider the following as a rule of a thumb:
+
+* In order to have the lowest possible latency, 2 vCPU are recommended per app 
+instance (k8s pod). The second vCPU will mostly be used by the GC and libuv
+threadpool. This will minimize the latency for your users, as well as the
+memory usage, as the GC will be run more frequently. The main thread won't
+have to stop to let the GC run.
```

Comment:
```suggestion
* In order to have the lowest possible latency, 2 vCPU are recommended per app 
instance (e.g., a k8s pod). The second vCPU will mostly be used by the garbage collector (GC) and libuv
threadpool. This will minimize the latency for your users, as well as the
memory usage, as the GC will be run more frequently. Also, the main thread won't
have to stop to let the GC run.
```